radio/src/targets/common/arm/stm32/stm32_spi.cpp

```diff
@@ -230,12 +230,27 @@ void stm32_spi_init(const stm32_spi_t* spi, uint32_t data_width, bool misoPullUp
   spiInit.TransferDirection = LL_SPI_FULL_DUPLEX;
   spiInit.Mode = LL_SPI_MODE_MASTER;
   spiInit.NSS = LL_SPI_NSS_SOFT;
+//  spiInit.NSS = LL_SPI_NSS_HARD_OUTPUT;
   spiInit.DataWidth = data_width;
 
+
   LL_SPI_Init(SPIx, &spiInit);
-  LL_SPI_Enable(SPIx);
+
 #if defined(STM32H7) || defined(STM32H7RS)
+  /* This locks SPI pins in AF mode when the SPI is disabled, to avoid glitches on CLK */
+  /* Temporary disable may be required to recover from weird states; see errata - pa01 */
+//  LL_SPI_EnableGPIOControl(SPIx);
+//  LL_SPI_DisableMasterRxAutoSuspend(SPIx);
+
+  /* Set transfer size = 0 (not using 'transfers') */
+  LL_SPI_SetTransferSize(SPIx, 0);
```

Comment:
According to RM00433 section 50.11.2, reset value of CR2 is 0, which means that this shouldn't be necessary.